src/atcore.cpp

```diff
@@ -228,5 +228,15 @@ void AtCore::stop()
 
 void AtCore::pushCommand(const QString &comm)
 {
-    serial()->pushCommand(plugin()->translate(comm));
+    if (!plugin()) {
```

Comment:
This is a solution, but not the beter one. The pointer was not initialized correctly, that's why the system was crashing.